kroxylicious-operator/DEV_GUIDE.md

```diff
@@ -44,23 +44,29 @@ minikube start --kubernetes-version=latest --driver=podman
 Building the operator distribution
 
 ```bash
-mvn package
+mvn -Pdist package
 ````
 
-should produce a directory matching `target/kroxylicious-operator-*-SNAPSHOT-bin`.
+should produce a directory matching `target/kroxylicious-operator-*-SNAPSHOT-app`.
+
+If you want to the skip the tests, then you can add `-DskipTests` to the above command, for example:
+
+```bash
+mvn -Pdist package -DskipTests
+````
 
 Build the operator image. For development purposes you can use the minikube registry directly using `minikube image build`, which will be faster than alternatives like pushing to quay.io from your host only for kube to pull the same image right back when you deploy the operator.
 
 ```bash
 KROXYLICIOUS_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:3.4.0:evaluate -Dexpression=project.version --quiet -DforceStdout)
-minikube image build -f Dockerfile.operator  .. -t quay.io/kroxylicious/operator:latest \
+minikube image build -f Dockerfile.operator .. -t quay.io/kroxylicious/operator:latest \
 --build-opt=build-arg=KROXYLICIOUS_VERSION=${KROXYLICIOUS_VERSION}
 ```
 
 ## Installing the operator
 
 ```bash
-kubectl apply -f install 
+kubectl apply -f kroxylicious-operator/target/packaged/install 
```

Comment:
FWIW... I think the kroxylicious-operator/DEV_GUIDE.md was written assuming the CWD was the kroxylicious-operator directory.